helpers/getSymbolDescription.js

```diff
@@ -5,26 +5,34 @@ var GetIntrinsic = require('../GetIntrinsic');
 var callBound = require('./callBound');
 
 var $SyntaxError = GetIntrinsic('%SyntaxError%');
+var getGlobalSymbolDescription = GetIntrinsic('%Symbol.keyFor%', true) || function () {};
+var thisSymbolValue = callBound('%Symbol.prototype.valueOf%', true);
 var symToStr = callBound('Symbol.prototype.toString', true);
 
 var getInferredName = require('./getInferredName');
 
-module.exports = function getSymbolDescription(symbol) {
-	if (!symToStr) {
+/* eslint-disable consistent-return */
+module.exports = callBound('%Symbol.prototype.description%', true) || function getSymbolDescription(symbol) {
+	if (!thisSymbolValue) {
 		throw new $SyntaxError('Symbols are not supported in this environment');
 	}
-	var str = symToStr(symbol); // will throw if not a symbol
+
+	// will throw if not a symbol primitive or wrapper object
+	var sym = thisSymbolValue(symbol);
```

Comment:
using `thisSymbolValue` isn't necessary, as `getInferredName` uses eval, and defers to the engine. I'd rather use symbolToString.

Reply:
This is necessary if we want to use `Symbol.keyFor`, as `Symbol.keyFor` will throw for `Symbol` wrapper objects.